### ISSUE TITLE
Paths with spaces not supported

### ISSUE THREAD

**Issue by holycrepe** [anthraxx/intellij-awesome-console#29]
Currently, a Windows-style path with a space is matched as separate paths. One possible fix for this is by allowing spaces in the URL pattern's `path` subgroup.  

See PR #27

**Comment by anthraxx** [comment]
hm this will create some performence problems with long lines, also its hard to catch all edge cases to determine how many space splitted parts are really a single file.
simply adding spaces into the matcher opens op tons of other problems.

**Comment by holycrepe** [comment]
Yes I agree. Perhaps we should allow for user-defined regex patterns?

**Comment by holycrepe** [comment]
For example, I modified my script's output to put brackets around a space, which allows me to define a custom regex pattern that will match that script's output without the issues you mentioned

**Comment by bluelovers** [comment]
![image](https://user-images.githubusercontent.com/167966/62434007-77bf8500-b769-11e9-831d-05e597d54a36.png)

hope can support space inside at a path like string start  and end with `" "`

**Comment by mahadi** [comment]
Any news on this. Actually this is a must have feature. At least wrapping in quotes should be supported.

**Comment by lasote** [comment]
Yes please, we need an "awesome/console/with path /with spaces/" support

**Comment by mahadi** [comment]
Any possibility to implement this as an opt-in for users who still need it (perhaps giving the option to specify that "" and '' paths with spaces are supported)? This would be really helpful.
